tests/testthat/test-Rules-methods.R

```diff
@@ -329,3 +329,65 @@ test_that("StoppingLowestDoseHSRBeta works correctly if first active dose is not
   )
   expect_identical(result, expected) # First active dose not applied.
 })
+
+test_that("NextBestInfTheory can be initiliazed ", {
+  expect_silent(NextBestInfTheory(target = 0.25, asymmetry = 0.1))
+})
+
+test_that("Helper function (h_info_theory_dist) for NextBestInfTheory is correct", {
+  # Values calculated using a different program.
+  p <- c(0.01, 0.2, 0.7)
+  gamma <- c(0.5, 0, 0.3)
+  a <- c(1, 1.8, 0.5)
+  expected <- c(24.25, 0.76, 1.16)
+  calculated <- h_info_theory_dist(p, gamma, a)
+  expect_equal(calculated, expected, tolerance = 0.01)
+})
+
+test_that("NextBestInfTheory produces consistent results", {
```

Comment:
this is nice in the sense that it looks at behavior inside study simulations - from a unit testing perspective we still need a test that looks at a single data set and what the `nextBest` method gives as result though (see my previous comment)

Reply:
@pc494 can you please add another test as mentioned here?